Dockerfile

```diff
@@ -0,0 +1,49 @@
+# An integration test & dev container which builds and installs libgdf & pygdf from master
+
+# Update based on your host's CUDA driver version
```

Comment:
The CUDA version of the container isn't dependent on the host CUDA version. The host doesn't even need CUDA installed. Let's change this comment to you can change this CUDA version based on your desired CUDA version for compatibility with other software potentially being added to the container.